assets/css/style.css

```diff
@@ -2234,3 +2234,12 @@ img.img-mouse {
 .team-v1 li > h3 {
 	color: #555;
 }
+
+/*Swagger Tables - Prevent Side Scrolling*/
+.parameter{
+  width: 100% !important;
+}
+
+.body-textarea {
+  width: 100% !important;
+}

```

Comment:
I'm very hesitant to use !important. I did a little digging into the swagger css. In swagger-ui/css/screen.css on line 462, what if we just change width from 300px to 200px? This would make our table slimmer, and get rid of the side scrolling issue.